r2/r2/lib/eventcollector.py

```diff
@@ -123,6 +123,12 @@ def vote_event(self, vote, old_vote=None,
             for name, value in private_data.iteritems():
                 event.add(name, value)
 
+        if context_data:
+            referrer_url = context_data.get('referer')
+            if referrer_url:
+                event.add("referrer_url", referrer_url)
```

Comment:
@jophuds @xiongchiamiov hrm.  I hadn't considered that as I hadn't noticed the constructor call.  I'll fix.

I'm inclined to just fix it to remove `http_referrer` and just have `referrer_url` and `referrer_domain` on all events when there's a `Referer` available.  This also means cleaning up the quarantine event to remove the resulting duplication.  I also don't feel strongly about it.